src/components/cap-volume.js

```diff
@@ -21,6 +22,65 @@ export default class CapVolume extends LitElement {
 		this.volumeData = {};
 	}
 
+	static styles = [
+		baseStyles,
+
+		css`
+			p,
+			ul {
+				font-family: var(--font-sans-text);
+
+				@media (min-width: 35rem) {
+					font-size: var(--font-size-175);
+				}
+			}
+
+			ul {
+				padding: 0;
+			}
+
+			li {
+				list-style-type: none;
+			}
+
+			a:link,
+			a:visited,
+			a:hover,
+			a:active {
+				text-decoration: none;
+				color: var(--color-blue-400);
+
+				&:hover {
+					color: var(--color-blue-500);
+					text-decoration: underline;
+				}
+			}
```

Comment:
I might have missed this in the reporter CSS — I made a mistake in the global.css file where I wrote css in a nested way like this. It doesn't throw an error because CSS is totally permissive of attempting to style selectors that don't exist, but we should rewrite it for clarity like this: 

```
	a:link, a:visited, a:hover, a:active {
		text-decoration: none;
		color: var(--color-blue-400);
         }
			
	a:hover {
        	color: var(--color-blue-500);
                 text-decoration: underline;
        }
```